_opensearch/rest-api/multi-search.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: default
+title: Multi search
+parent: REST API reference
+nav_order: 130
+---
+
+# Multi search
+Introduced 1.0
+{: .label .label-purple }
+
+The multi-search operation lets you bundle multiple search requests and send them to your OpenSearch cluster in a single request. This operation executes searches in parallel, so you get back the response more quickly as compared to independent search requests. It also executes each request independently, so the failure of one request doesn't affect the others.
+
+The multi-search request body follows this pattern:
+
+```
+header\n
+body\n
+header\n
+body\n
+```
+
+OpenSearch uses newline characters to parse multi-search requests and requires that each request ends with a newline character.
+
+## Example
+
+```json
+GET _msearch
+{"index":"opensearch_dashboards_sample_data_logs"}
+{"query":{"match_all":{}},"from":0,"size":10}
+{"index":"opensearch_dashboards_sample_data_ecommerce","search_type":"dfs_query_then_fetch"}
+{"query":{"match_all":{}}}
+```
+
+## Path and HTTP methods
+
+```
+GET <target>/_msearch
+```
+
+## URL parameters
+
+All multi-search URL parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`allow_no_indices` | Boolean | Whether to ignore wildcards that don't match any indices. Default is `true`.
+`css_minimize_roundtrips` | Boolean | If true, network roundtrips between the local node and remote clusters are minimized for cross-cluster search requests. Default is `true`.
+`expand_wildcards` | Enum | Expands wildcard expressions to concrete indices. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
+`ignore_unavailable` | Boolean | If an index from the indices list doesn’t exist, whether to ignore it rather than fail the query. Default is `false`.
+`max_concurrent_searches` | Integer | Maximum number of searches executed in parallel. Default is `max(1, (number of of data nodes * min(search thread pool size, 10)))`.
+`max_concurrent_shard_requests` | Integer | Maximum number of concurrent shard requests that each sub-search request executes per node. Default is 5. If you have an environment where a very low number of concurrent search requests are expected, a higher value of this parameter might improve performance.
```

Comment:
I think this should be "...a very low number of concurrent search requests is expected..."

Reply:
Fixed.